ttnn/cpp/ttnn/operations/matmul/device/matmul_op.hpp

```diff
@@ -7,7 +7,8 @@
 
 #include "tensor/tensor.hpp"
 
-#include "tt_dnn/op_library/eltwise_unary/eltwise_unary_op.hpp"
+// #include "tt_dnn/op_library/eltwise_unary/eltwise_unary_op.hpp"
```

Comment:
Why is the include commented out instead of deleted?

Reply:
@bbradelTT will safely remove these when all CI pipelines are green.

Reply:
Ok. Thanks.